modules/virt-importing-vm-wizard.adoc

```diff
@@ -72,13 +72,22 @@ ifdef::virt-importing-rhv-vm[]
 . Optional: You can select *Start virtual machine on creation*.
 endif::[]
 
-. Optional: Click *Edit* to update the following settings:
+. Click *Edit* to update the following settings:
 
 ifdef::virt-importing-rhv-vm[]
 * *General* -> *Name*: The VM name is limited to 63 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
 * *General* -> *Description*: Optional description of the VM.
 * *Storage* -> *Storage Class*: The default storage class must be NFS.
 * *Networking* -> *Network*: You can select a network from a list of available `NetworkAttachmentDefinition` objects.
+
```

Comment:
This paragraph should be under storage > Storage class above, as this refer to Editing the disk under it.

Reply:
What are you referring to? All the options or just "networking"? 

I did not put the block storage procedure under storage because it requires its own procedure (clicking the kebab menu, clicking Advanced, etc.) It's too many steps to put in a bullet list. That's why the block storage configuration is a separate step.

Reply:
OK. I understand. thanks for explaining.